legend-engine-xts-dataquality/legend-engine-xt-dataquality-protocol/src/main/java/org/finos/legend/engine/protocol/dataquality/metamodel/DataSpaceDataQualityExecutionContext.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package org.finos.legend.engine.protocol.dataquality.metamodel;
+
+import org.finos.legend.engine.protocol.pure.v1.model.context.PackageableElementPointer;
+
+public class DataSpaceDataQualityExecutionContext extends DataQualityExecutionContext
+{
+    public String _type = "dataSpaceDataQualityExecutionContext";
```

Comment:
we shouldnt really carry this in the class. This should just be used to serialize/deseralize to json.

Reply:
_type is the property used in json subtypes for serialization and deserialization [DataQualityExecutionContext](https://github.com/goldmansachs/legend-engine/blob/review-new-data-quality-module/legend-engine-xts-dataquality/legend-engine-xt-dataquality-protocol/src/main/java/org/finos/legend/engine/protocol/dataquality/metamodel/DataQualityExecutionContext.java#L21). I think its needed here

this is similar to [RootGraphFetchTree](https://github.com/goldmansachs/legend-engine/blob/af6f1f9522b9ea36fbf388b447bd20295a756422/legend-engine-core/legend-engine-core-base/legend-engine-core-language-pure/legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/valueSpecification/raw/classInstance/graph/RootGraphFetchTree.java#L21)